Symfony/CS/Tests/Tokenizer/TokensAnalyzerTest.php

```diff
@@ -38,8 +38,14 @@ public function bar4()
 
         return " ({$a})";
     }
+
     public function bar5($data)
     {
+        $message = $data;
+        $example = function ($arg) use ($message) {
+            echo $arg . ' ' . $message;
+        };
+        $example('hello');
```

Comment:
This still shows in the diff because github is auto-resolving merge conflicts. Only when this PR is actually rebased will the already changed code dissapear.